### ISSUE TITLE
Android a11y text input

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#12785]
- [x] A text field doesn't identify itself as EditBox 
- [x] Text field doesn't give hint to "double tap to enter text" (instead says "double tap to activate")
- [x] A text field that currently has the blinking text cursor is not announced as "editing"
- [x] The text in a text field is not announced when a11y focused
- [ ] Letters entered via keyboard are not echoed back
- [ ] After typing a space the last word is not echoed back
- [ ] Cannot move curser with volume keys
- [ ] Local TalkBack menu (swiping up and right) is empty (doesn't over curser control, etc)

**Comment by goderbauer** [comment]
Related: iOS text input (https://github.com/flutter/flutter/issues/12786)

**Comment by goderbauer** [comment]
The first view problems can probably be fixed by setting the "class" property of the `AccessibilityInfoNode` responsible for the text field to "EditText".

**Comment by goderbauer** [comment]
Confirmed:
* 1+2 can be fixed with setting class to "EditText".
* 3 can be fixed by properly setting [setFocused](https://developer.android.com/reference/android/view/accessibility/AccessibilityNodeInfo.html#setFocused(boolean)).
* 4 can be fixed by exposing the content of the text field to the semantics tree as "value" (via describeSemanticsConfiguration)

**Comment by goderbauer** [comment]
> The text in a text field is not announced when a11y focused
> After typing a space the last word is not echoed back

This most likely requires sending properly crafted [TYPE_VIEW_TEXT_CHANGED](https://developer.android.com/reference/android/view/accessibility/AccessibilityEvent.html#TYPE_VIEW_TEXT_CHANGED) events.

**Comment by goderbauer** [comment]
> Cannot move curser with volume keys.

Need to implement [ACTION_PREVIOUS_AT_MOVEMENT_GRANULARITY](https://developer.android.com/reference/android/view/accessibility/AccessibilityNodeInfo.html#ACTION_PREVIOUS_AT_MOVEMENT_GRANULARITY) and [ACTION_NEXT_AT_MOVEMENT_GRANULARITY](https://developer.android.com/reference/android/view/accessibility/AccessibilityNodeInfo.html#ACTION_NEXT_AT_MOVEMENT_GRANULARITY).
